users/dhertz/dhertz.c

```diff
@@ -0,0 +1,102 @@
+#include "dhertz.h"
+#include "quantum.h"
```

Comment:
Redundant.  If the dhertz.h has it inclueded, it's not included here.

Same is likely true of action.h (quantum.h includes it, IIRC)